install.yaml

```
@@ -22,3 +22,6 @@ post_install_actions:
   if ( test -f "$HOME/.ddev/commands/web/browsersync" ) ; then
     echo "Note: '~/.ddev/commands/web/browsersync' is no longer required and can be safely removed." && exit 0
   fi
+  #ddev-nodisplay
+  #ddev-description:Remove old 'docker-compose.browsersync.yaml'
+  if grep "#ddev-generated" $DDEV_APPROOT/.ddev/docker-compose.browsersync.yaml 2>/dev/null; then rm "$DDEV_APPROOT/.ddev/docker-compose.browsersync.yaml"; fi
```

It's always good to have a force flag for disruptive operations.
This is needed because people may have an `alias rm="rm -i"` on their host (I'm one of those people) which prevents accidental deletion.

```suggestion
  if grep "#ddev-generated" $DDEV_APPROOT/.ddev/docker-compose.browsersync.yaml 2>/dev/null; then rm -f "$DDEV_APPROOT/.ddev/docker-compose.browsersync.yaml"; fi
```

---

Good call.